src/functions.php

```diff
@@ -356,11 +356,11 @@ function xml_adopt(SimpleXMLElement $to, SimpleXMLElement $from)
 /**
  * Upload cards to fritzbox
  *
- * @param string $xml
- * @param array $config
+ * @param SimpleXMLElement  $xmlPhonebook
+ * @param array             $config
  * @return void
  */
-function upload(string $xml, $config)
+function uploadPhonebook(SimpleXMLElement $xmlPhonebook, array $config)
```

Comment:
I don't think that inside an `uploadPhonebook` function its really necessary to have an xmlPhonebook variable. But thats just personal preference, as you want.

Reply:
Would keep it... I like "speaking" params - especially inside cool IDEs...